src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -328,10 +368,13 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
 
         ...(primary
```

Comment:
I think this would be greatly simplified (easier to redline & maintain) if we can remove `primary` prop and use just the `color` prop, since in Teams world `primary` is synonymous with just using brand color anyway. I think having `primary` prop makes sense on button and maybe some other components, but not for menu.